packages/loader/container-loader/src/container.ts

```diff
@@ -753,12 +753,21 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return this.protocolHandler.quorum;
     }
 
+    public dispose?(error?: ICriticalContainerError) {
```

Comment:
@reviewers Is there a better way of doing this? Having it required broke type testing

Reply:
I think we're allowed to break the implementation of Container in a minor bump.  Could be wrong.  @anthony-murphy ?

Reply:
What I mean is, this would show up as breaking forward-compat (old matches new), but not backwards-compat (new matches old). i.e. if someone is directly using or extending `Container` and then this starts showing up, their code still works. And of course it still meets the interface, which is even more important.

Reply:
From internal Teams thread: making the method required breaks forward compat. Most likely doesn't affect anything in this case, but better to be safe than sorry